test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -809,6 +848,14 @@ func runWorkload(b *testing.B, tc *testCase, w *workload) []DataItem {
 				}
 			}
 
+		case *sleepOp:
+			timer := time.NewTimer(concreteOp.Duration)
+			select {
+			case <-ctx.Done():
+				return nil
+			case <-timer.C:
```

Comment:
nit: We can eliminate L852 by placing a time.After():

```go
case <-time.After(concreteOp.Duration):
```

Reply:
nice idea, thanks.